src/outpack/filestore.py

```diff
@@ -30,7 +30,7 @@ def exists(self, hash):
         return os.path.exists(self.filename(hash))
 
     def put(self, src, hash):
-        hash_validate(src, hash)
+        hash_validate_file(src, hash)
```

Comment:
Renamed this to `hash_validate_file` as I've added a separate `hash_validate_string` which both call `hash_validate`